Proposal - Adjust the BAT-A Debt Ceiling - Sepember 17, 2020.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Increase the BAT-A Debt Ceiling - September 17, 2020
+summary: Signal your support to increase the BAT Debt Ceiling to 10 million dai
+discussion_link: https://forum.makerdao.com/t/signal-request-adjust-the-bat-debt-ceiling/3919/3
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: BAT Debt Ceiling Increase - September 17, 2020
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Governance Community. The community can use this poll to express support or opposition for a **BAT Debt Ceiling** increase to 10 million dai.
+
+This Governance Poll ([FAQ](https://community-development.makerdao.com/makerdao-scd-faqs/scd-faqs/governance)) will be active for three days beginning on Monday, September 21 at 16:00 UTC, the results of which may inform an Executive Vote which will go live on Friday, September 25, at 16:00 UTC.
+
+## Review
+
+The **BAT Debt Ceiling** was voted on in a forum signal request. Please review the [Thread](https://forum.makerdao.com/t/signal-request-adjust-the-bat-debt-ceiling/3919/3) to inform your position before voting.
+
+## Next Steps
+
+If the winning result of this poll is Yes, then the **BAT Debt Ceiling** parameter value in the Maker Protocol change to 10 million dai will be included in the Executive Vote on the Friday following the conclusion of this poll. 
```

Comment:
I think it's probably easier to just keep this generic, I don't think it's worthwhile restating the poll. "If this poll ends with a 'Yes' majority, this change will be included in the Executive Vote on..."